libs/books/feature/src/lib/book-search/books-grid/books-grid-item/books-grid-item.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SharedTestingModule } from '@tmo/shared/testing';
+import { BooksFeatureModule } from '../../../books-feature.module';
+import { BooksGridItemComponent } from './books-grid-item.component';
+
+describe('BooksGridItemComponent', () => {
+  let component: BooksGridItemComponent;
+  let fixture: ComponentFixture<BooksGridItemComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [BooksFeatureModule, SharedTestingModule]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(BooksGridItemComponent);
+    component = fixture.componentInstance;
+    component.book = {
+      id: `${Date.now()}b`,
+      title: `Some Title ${Math.floor(Math.random() * 9999)}b`,
+      authors: ['a', 'b', 'c'],
```

Comment:
You've gone to some trouble here to randomise the book details but they don't seem to come into play. Any reason?

Reply:
added to the component test to check that the book is as expected